frame/democracy/src/lib.rs

```diff
@@ -831,11 +828,25 @@ pub mod pallet {
 		/// Weight: `O(1)`
 		#[pallet::weight(T::WeightInfo::fast_track())]
 		pub fn fast_track(
-			_origin: OriginFor<T>,
+			origin: OriginFor<T>,
 			proposal_hash: T::Hash,
 			voting_period: T::BlockNumber,
 			delay: T::BlockNumber,
 		) -> DispatchResult {
+			let maybe_ensure_instant = if voting_period < T::FastTrackVotingPeriod::get() {
```

Comment:
This comes from https://github.com/liberland/liberland_substrate/commit/8e766d3969ad5911142e21bd20bd5e4b72197ed2#diff-7d396d7ca3beeacb0775a3a5de148508e6d8f4640e0eacd728c00f71a2268a9fL833-R849